templates/components/cart/shipping-estimator.html

```diff
@@ -67,5 +67,5 @@
             <button class="button button--primary button--small shipping-estimate-submit">{{lang 'cart.shipping_estimator.estimate_shipping'}}</button>
         </dl>
     </form>
-    <div class="shipping-quotes"></div>
+    <div class="shipping-quotes" role="alert" aria-atomic=true aria-live="assertive"></div>
```

Comment:
I believe we should use aria-atomic="true" instead of aria-atomic=true. Attribute value should be in quotes

Reply:
done